app/assets/stylesheets/application.scss

```
@@ -3,4 +3,10 @@
  *= require_self
  */
 
- @import "bootstrap/scss/bootstrap";
\ No newline at end of file
+.base-container {
+  max-width: 992px;
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+@import "bootstrap/scss/bootstrap";
```

今後影響が出る可能性がありますので，`@import` は一番上に移動させて下さい。
（`require_self` などよりは下に移動して下さい）

```suggestion
@import "bootstrap/scss/bootstrap";

.base-container {
  max-width: 992px;
  margin: 0 auto;
  padding: 1rem;
}
```